src/actions/guides/database/deleting_records.cr

```diff
@@ -13,33 +13,52 @@ class Guides::Database::DeletingRecords < GuideAction
     Similar to the `SaveOperation`, Avram comes with a `DeleteOperation` that's generated with each model.
     This allows you to write more complex logic around deleteing records. (e.g. delete confirmations, etc...)
 
-    ### Setup
+    ### Simple deletes
 
-    These classes go in your `src/operations/` directory, and will inherit from `{ModelName}::DeleteOperation`.
+    If you just want to delete a record without any validations or callbacks, the simplest way is to use the generated `{ModelName}::DeleteOperation`
+
+    For example, if we have a `Server` model, Lucky will generate a `Server::DeleteOperation` that you can use:
+
+    ```crystal
+    server = ServerQuery.find(123)
+    Server::DeleteOperation.destroy!(server)
+    ```
+
+    If the record fails to be deleted, an `Avram::InvalidOperationError` will be raised.
+
+    ### Setting up a custom DeleteOperation
+
+    You can customize DeleteOperations with callbacks and validations.  These
+    classes go in your `src/operations/` directory, and will inherit from
+    `{ModelName}::DeleteOperation`.
 
     ```crystal
     # src/operations/delete_server.cr
     class DeleteServer < Server::DeleteOperation
     end
     ```
 
-    ### Using in actions
+    ### Using a `DeleteOperation` in actions
 
     The interface should feel pretty familiar. The object being deleted is passed in to the `delete` method, and a block will
     return the operation instance, and the object being deleted.
 
+    > Note: The second block argument (the deleted object) is optional
+
     ```crystal
     # src/actions/servers/delete.cr
     class Servers::Delete < BrowserAction
       delete "/servers/:server_id" do
         server = ServerQuery.find(server_id)
 
-        DeleteServer.delete(server) do |operation, deleted_server|
+        # The second block argument can be completely removed.
+        # It is shown here with a `_` to show that it can be used if needed.
```

Comment:
Though, this is true, it doesn't fit in this case because in the `else` branch, the 2nd arg is used.